ovs/proto_trace.go

```diff
@@ -80,12 +81,39 @@ func (df *DataPathFlows) UnmarshalText(b []byte) error {
 		return errors.New("error unmarshalling text, no comma delimiter found")
 	}
 
-	// first string is always the protocol
-	df.Protocol = Protocol(matches[0])
+	for _, match := range matches {
+		switch Protocol(match) {
+		case ProtocolARP:
+			fallthrough
+		case ProtocolICMPv4:
+			fallthrough
+		case ProtocolICMPv6:
+			fallthrough
+		case ProtocolIPv4:
+			fallthrough
+		case ProtocolIPv6:
+			fallthrough
+		case ProtocolTCPv4:
+			fallthrough
+		case ProtocolTCPv6:
+			fallthrough
+		case ProtocolUDPv4:
+			fallthrough
+		case ProtocolUDPv6:
+			df.Protocol = Protocol(match)
+			continue
```

Comment:
If the intention with all the `fallthrough` is to make all those Protocols have the same action, you can also combine all of them into the same case separated by `,`, e.g.
```
case ProtocolARP, ProtocolICMPv4, ProtocolIPv4:
   df.Protocol = Protocol(match)
   continue
```